[ISSUE 339] Refactor 'About Yoroi' items

ScaredIbis opened the pull request:
For issue https://github.com/Emurgo/yoroi-frontend/issues/339

This change introduces a shared component (LinkButton), used directly in AboutYoroiSettings. The FooterItem component has also been refactored and uses LinkButton as a child. 

This has allowed styles used for hiding text on small screens to be separated from the AboutYoroiSettings component.

BEFORE:
![image](https://user-images.githubusercontent.com/14157626/56470853-786c2e00-648e-11e9-898c-c46fc31f7563.png)


AFTER:
![image](https://user-images.githubusercontent.com/14157626/56471638-23351a00-6498-11e9-8524-fb0334373bb6.png)

SebastienGllmt commented:
Hey, thanks for the PR!
Three things to mention:

1) We just introduced a new theme so make sure this works on both themes (you can switch themes from the settings page). I fixed the merge conflict for you.
1) I noticed your PR adds a scrollbar on the LinkButton (as seen in your screenshot). That needs to be fixed.
1) The reason the media query was added was so that either all buttons show text or none show text. Looks like your PR breaks this (in both the wallet add screen and the About Yoroi screen)

ScaredIbis commented:
I have addressed the issues you brought up:

1. I made some minor style changes so that it functions the same and looks decent in both themes.
2. I have updated the styles for HorizontalFlexContainer (shared component) to remove the scrollbar that was showing up. From what I can see, the style update doesn't break other components that also use HorizontalFlexContainer.
3.  The media queries are now applying to both Wallet Creation and About Yoroi pages properly. These styles are defined in a way that make it easy to change the breakpoints for each page individually.

SebastienGllmt commented:
Looks great! Thanks!
